front/pages/about.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+  <layout>
+    <div class="min-vh100-fh">
+      TODO About page
+    </div>
+
+    <main-footer />
+  </layout>
+</template>
+
+<script>
+import Layout from '@/components/layout/Layout'
+import MainFooter from '@/components/layout/MainFooter'
+
+export default {
+  components: {
+    Layout,
+    MainFooter,
```

Comment:
trailing commas. its actually eslint warning so please fix eslint warnings  in your pr

Reply:
I'd actually suggest to edit eslint to add trailing commas, having them is better for git is it will affect only one line if you add something to list instead of two line, and it's just simpler to code with trailing commas on multiline, not sure why it didn't go to general standard, trailing comms is basically a standard now for any languages as it just makes sense https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8

Reply:
How is it standard when in one of the most common used eslint config its error? E.g. in eslint config standard. We are just using default nuxt eslint config for the sake of uniformity, rather than adjusting rules on the fly. Also if u check vue docs there wont be any dangling commas in example. So no, i don't see how its standard now if its not in the official docs and its a error in one of the most used eslint config. While i see the rationale behind it, i just stick to standard thing which is no trailing comma in nuxt eslint preset

Reply:
Ok, no point of arguing about it, that standard is more about just following common standard by all than actual convenience and beauty of code. I personally always enforce comma dangle in my projects as it just makes more sense. I'll add autofixer to my IDE.